samples/app-sso/csharp/app_sso/SimpleGraphClient.cs

```diff
@@ -0,0 +1,215 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Microsoft.Graph;
+
+namespace Microsoft.BotBuilderSamples
+{
+    // This class is a wrapper for the Microsoft Graph API
+    // See: https://developer.microsoft.com/en-us/graph
+    public class SimpleGraphClient
+    {
+        private readonly string _token;
+
+        public SimpleGraphClient(string token)
+        {
+            if (string.IsNullOrWhiteSpace(token))
+            {
+                throw new ArgumentNullException(nameof(token));
+            }
+
+            _token = token;
+        }
+
+        // Sends an email on the users behalf using the Microsoft Graph API
+        public async Task SendMailAsync(string toAddress, string subject, string content)
+        {
```

Comment:
SendMailAsync method is not being used anywhere in this sample app, Please cleanup the unused code